src/object-store/src/object_store_manager.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use snafu::ensure;
+
+use crate::error::Result;
+use crate::{error, ObjectStore};
+
+#[derive(Clone)]
+pub struct ObjectStoreManager {
+    stores: HashMap<String, ObjectStore>,
+    global_object_store: String,
```

Comment:
question(non-blocking): What about `default_object_store`? I think it acts like a default storage. e.g. It manages object stores by names and provides a global (default) object store.